src/filterData.js

```diff
@@ -11,10 +11,11 @@ const sortByIntOrdre = (a, b) => {
 const numify = id => parseInt(id.replace(/^KALIARTI/, ""));
 
 const isValidSection = node => node.etat !== "ABROGE" && node.etat !== "PERIME";
+
 // the API returns all the version of a given article. we pick the latest one
-const isValidArticle = (currentArticle, index, articles) => {
+const isValidArticleFilter = (currentArticle, index, articles) => {
```

Comment:
I found  isValidArticleFilter not very meaningFull (I initially tought it should return true / false) 
does `latestArticleVersionFilter` sounds a bit clearer ?